<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Centralwindow</class>
 <widget class="QMainWindow" name="Centralwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account Details</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/images/wmremove-transformed.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="depositbtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 12pt &quot;Microsoft Sans Serif&quot;;
background-color:#2b547d;
color:white;
border-radius:20px;
padding:7px 14px;
}
QPushButton:hover{
background-color:rgb(41, 128, 185);
}
QPushButton:pressed{
background-color:rgb(31, 97, 141);
}</string>
    </property>
    <property name="text">
     <string>deposit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="withdrawbtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 12pt &quot;Microsoft Sans Serif&quot;;
background-color:#2b547d;
color:white;
border-radius:20px;
padding:7px 14px;
}
QPushButton:hover{
background-color:rgb(41, 128, 185);
}
QPushButton:pressed{
background-color:rgb(31, 97, 141);
}</string>
    </property>
    <property name="text">
     <string>withdraw</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transferbtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 12pt &quot;Microsoft Sans Serif&quot;;
background-color:#2b547d;
color:white;
border-radius:20px;
padding:7px 14px;
}
QPushButton:hover{
background-color:rgb(41, 128, 185);
}
QPushButton:pressed{
background-color:rgb(31, 97, 141);
}</string>
    </property>
    <property name="text">
     <string>transfer Money</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transactionbtn">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 12pt &quot;Microsoft Sans Serif&quot;;
background-color:#2b547d;
color:white;
border-radius:20px;
padding:7px 14px;
}
QPushButton:hover{
background-color:rgb(41, 128, 185);
}
QPushButton:pressed{
background-color:rgb(31, 97, 141);
}</string>
    </property>
    <property name="text">
     <string>transaction history</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:white;
font: 700 20pt &quot;Arial&quot;;
color:#2b547d;}</string>
    </property>
    <property name="text">
     <string> ACCOUNT DETAILS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-9</y>
      <width>731</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:rgb(199, 201, 213);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>511</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:rgb(255, 255, 255);
border-radius:10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>211</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color:#2b547d;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:#2b547d;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/images/wmremove-transformed.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/images/wmremove-transformed.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>430</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:#2b547d;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/new/prefix1/images/wmremove-transformed.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="username_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Sans Serif</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 14pt &quot;Microsoft Sans Serif&quot;;
background-color: rgb(255, 255, 255);
color:rgb(43, 84, 125);
border:2px solid rgb(43, 84, 125);
padding:5px;
}
QLabel:hover {
    border: 2px solid rgb(0, 120, 215); /* Light blue */
	background-color: rgb(240, 248, 255);
}

/* Change border and background when focused (clicked) */
QLabel:focus {
    border: 2px solid rgb(0, 150, 255); /* Brighter blue */
    background-color: rgb(230, 247, 255); /* Light blue background */
}
</string>
    </property>
    <property name="text">
     <string>Username: </string>
    </property>
   </widget>
   <widget class="QLabel" name="email_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 14pt &quot;Microsoft Sans Serif&quot;;
background-color: rgb(255, 255, 255);
color:rgb(43, 84, 125);
border:2px solid rgb(43, 84, 125);
padding:5px;
}
QLabel:hover {
    border: 2px solid rgb(0, 120, 215); /* Light blue */
	background-color: rgb(240, 248, 255);
}

/* Change border and background when focused (clicked) */
QLabel:focus {
    border: 2px solid rgb(0, 150, 255); /* Brighter blue */
    background-color: rgb(230, 247, 255); /* Light blue background */
}
</string>
    </property>
    <property name="text">
     <string>Email: </string>
    </property>
   </widget>
   <widget class="QLabel" name="account_no_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 14pt &quot;Microsoft Sans Serif&quot;;
background-color: rgb(255, 255, 255);
color:rgb(43, 84, 125);
border:2px solid rgb(43, 84, 125);
padding:5px;
}
QLabel:hover {
    border: 2px solid rgb(0, 120, 215); /* Light blue */
	background-color: rgb(240, 248, 255);
}

/* Change border and background when focused (clicked) */
QLabel:focus {
    border: 2px solid rgb(0, 150, 255); /* Brighter blue */
    background-color: rgb(230, 247, 255); /* Light blue background */
}
</string>
    </property>
    <property name="text">
     <string>Account Number: </string>
    </property>
   </widget>
   <widget class="QLabel" name="acc_type_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 14pt &quot;Microsoft Sans Serif&quot;;
background-color: rgb(255, 255, 255);
color:rgb(43, 84, 125);
border:2px solid rgb(43, 84, 125);
padding:5px;
}
QLabel:hover {
    border: 2px solid rgb(0, 120, 215); /* Light blue */
	background-color: rgb(240, 248, 255);
}

/* Change border and background when focused (clicked) */
QLabel:focus {
    border: 2px solid rgb(0, 150, 255); /* Brighter blue */
    background-color: rgb(230, 247, 255); /* Light blue background */
}
</string>
    </property>
    <property name="text">
     <string>Account Type: </string>
    </property>
   </widget>
   <widget class="QLabel" name="balance_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
font: 14pt &quot;Microsoft Sans Serif&quot;;
background-color: rgb(255, 255, 255);
color:rgb(43, 84, 125);
border:2px solid rgb(43, 84, 125);
padding:5px;
}
QLabel:hover {
    border: 2px solid rgb(0, 120, 215); /* Light blue */
	background-color: rgb(240, 248, 255);
}

/* Change border and background when focused (clicked) */
QLabel:focus {
    border: 2px solid rgb(0, 150, 255); /* Brighter blue */
    background-color: rgb(230, 247, 255); /* Light blue background */
}
</string>
    </property>
    <property name="text">
     <string>Initial Balance: </string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_13</zorder>
   <zorder>label_5</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_11</zorder>
   <zorder>depositbtn</zorder>
   <zorder>withdrawbtn</zorder>
   <zorder>transferbtn</zorder>
   <zorder>transactionbtn</zorder>
   <zorder>label_6</zorder>
   <zorder>label_15</zorder>
   <zorder>label_7</zorder>
   <zorder>label_16</zorder>
   <zorder>username_label</zorder>
   <zorder>email_label</zorder>
   <zorder>account_no_label</zorder>
   <zorder>acc_type_label</zorder>
   <zorder>balance_label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
